### ISSUE TITLE
Installation instructions produce package resolve error

### ISSUE THREAD

**Issue by hanneskaeufler** [tailwindlabs/tailwindcss.com#675]
### Describe the problem:

I tried to start a next.js project as per [instructions](https://nextjs.org/docs/getting-started).

`mkdir foo && cd foo && npx create-next-app .`

That installs cleanly

```
Success! Created foo at /Users/hanneskaeufler/Projects/personal/foo
```

Now on to installing tailwind as per [instructions](https://tailwindcss.com/docs/guides/nextjs)

`npm install tailwindcss postcss autoprefixer`

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! Found: postcss@8.1.7
npm ERR! node_modules/postcss
npm ERR!   postcss@"8.1.7" from next@10.0.3
npm ERR!   node_modules/next
npm ERR!     next@"10.0.3" from the root project
npm ERR!   postcss@"*" from the root project
npm ERR!   3 more (autoprefixer, tailwindcss, postcss-js)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer postcss@"^8.1.13" from postcss-nested@5.0.3
npm ERR! node_modules/tailwindcss/node_modules/postcss-nested
npm ERR!   postcss-nested@"^5.0.1" from tailwindcss@2.0.2
npm ERR!   node_modules/tailwindcss
npm ERR!     tailwindcss@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/hanneskaeufler/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/hanneskaeufler/.npm/_logs/2020-12-13T12_39_18_787Z-debug.log
```

```
~/Projects/personal/foo (master) node --version
v15.4.0
~/Projects/personal/foo (master) npm --version
7.0.15
~/Projects/personal/foo (master)
```

### Link to a minimal reproduction:

This one-liner reliably errors out for me:

```
mkdir foo && cd foo && npx create-next-app . && npm install tailwindcss postcss autoprefixer
```

**Comment by hanneskaeufler** [comment]
> Hey! Thank you for your bug report!
> Much appreciated! 🙏
> 
> It looks like that you installed one of those dependencies in the past and that you have an incorrect version cached.
> Things you can do:
> 
> ```shell
> mkdir foo && cd foo && npx create-next-app . && npm install tailwindcss@latest postcss@latest autoprefixer@latest
> ```
> 
> > Note the `@latest` tags
> 
> I'll PR a change that ensures this behaviour, I'll close this issue once merged!

That did the trick, thanks so much for the quick fix and help. Rock on ✌️

**Comment by akojimsg** [comment]
Ran into a similar problem while doing the getting tutorial on next.sj. Command to install tailwindcss is:

````
npm install tailwindcss postcss-preset-env postcss-flexbugs-fixes
````
 I was able to fix the problem by following these steps:

1. Delete __node_modules__ directory and run `npm i`
2. As the error message indicates, `npm i tailwindcss postcss-preset-env postcss-flexbugs-fixes
 --force`

The first step can also be executed with the `--force` option.
